server/db/seeds.rb

```diff
@@ -33,23 +33,17 @@
     definition: 'An activator interaction is when a drug activates a biological response from a target, although the mechanism by which it does so may not be understood.',
     reference: 'DrugBank examples: <a href="http://www.ncbi.nlm.nih.gov/pubmed/12070353">pubmed:12070353</a>'
   },
-  {
-    type: 'n/a',
-    directionality: nil,
-    definition: 'DGIdb assigns this label to any drug-gene interaction for which the interaction type is not specified by the reporting source.',
-    reference: nil
-  },
   {
     type: 'negative modulator',
     directionality: 1,
     definition: 'In a negative modulator interaction, the drug negatively regulates the amount or activity of its target. In contrast to an inhibitory allosteric modulator, this interaction type may not involve any direct binding to the target.',
     reference: '<a href="https://en.wikipedia.org/wiki/Allosteric_modulator">Wikipedia - Allosteric Modulator</a>'
   },
   {
-    type: 'partial agonist',
-    directionality: 0,
-    definition: 'In a partial agonist interaction, a drug will elicit a reduced amplitude functional response at its target receptor, as compared to the response elicited by a full agonist.',
-    reference: '<a href="https://en.wikipedia.org/wiki/Receptor_antagonist#Partial_agonists">Wikipedia - Receptor Antagonist</a>'
+    type: 'antibody',
+    directionality: 1,
+    definition: 'TODO',
+    reference: 'TODO'
```

Comment:
@mcannon068nw input pls

Reply:
type: 'antibody',
    directionality: 1,
    definition: 'An antibody interaction occurs when an antibody drug specifically binds the target molecule. Most interactions of this class fall under various natural antibody functions: neutralization, antibody-dependent cell-mediated cytotoxicity (ADCC), complement-dependent cytotoxicity (CDC), or drug delivery carriers.', 
    reference: '<a href="https://www.ncbi.nlm.nih.gov/pubmed/26441475/">pubmed:26441475</a>'